shells/dev/webpack.config.js

```diff
@@ -54,6 +54,7 @@ module.exports = {
           {
             loader: 'css-loader',
             options: {
+              sourceMap: true,
```

Comment:
Missed this. It's pretty nice, you easily find the original style in the chrome sources panel:

![screen shot 2017-06-29 at 7 31 15 am](https://user-images.githubusercontent.com/590904/27685598-f27cf2b6-5c9c-11e7-9b32-b10215f247d1.png)